wallet/src/de/schildbach/wallet/ui/notifications/NotificationManagerWrapper.kt

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2022 Dash Core Group.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package de.schildbach.wallet.ui.notifications
+
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import androidx.core.app.NotificationCompat
+import androidx.core.app.TaskStackBuilder
+import dagger.hilt.android.qualifiers.ApplicationContext
+import de.schildbach.wallet.Constants
+import de.schildbach.wallet_test.R
+import org.dash.wallet.common.services.NotificationService
+import javax.inject.Inject
+
+class NotificationManagerWrapper @Inject constructor(
```

Comment:
`NotificationManager` can be injected directly, but if we ever decide to use some in-app notification mechanism instead of system notifications, this will allow making changes easily.